lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -19,25 +19,10 @@ def supports_term?
         term_config.present?
       end
 
-      # Return term url encoding defined in the configuration for this authority.
-      # @return [Hash] the configured term url
-      def term_url
-        Config.config_value(term_config, :url)
-      end
-
       # Return term url template defined in the configuration for this authority.
-      # @return [String] the configured term url template
-      def term_url_template
-        Config.config_value(term_url, :template)
-      end
-
-      # Return term url parameter mapping defined in the configuration for this authority.
-      # @return [Hash] the configured term url parameter mappings with variable name as key
-      def term_url_mappings
-        return @term_url_mappings unless @term_url_mappings.nil?
-        mappings = Config.config_value(term_url, :mapping)
-        return {} if mappings.nil?
-        Hash[*mappings.collect { |m| [m[:variable].to_sym, m] }.flatten]
+      # @return [Qa::IriTemplate::UrlConfig] the configured term url template
+      def url_config
+        @url_config ||= Qa::IriTemplate::UrlConfig.new(term_config[:url]) if supports_term?
```

Comment:
The Qa::IriTemplate::UrlConfig class processes everything, so #term_url, #term_url_template, and #term_url_mappings are obsolete. Removed these methods and their tests.